test/src/main/java/io/strimzi/test/k8s/HelmClient.java

```diff
@@ -48,6 +48,7 @@ public HelmClient install(Path chart, String releaseName, Map<String, Object> va
                 releaseName,
                 "--set", values,
                 "--timeout", INSTALL_TIMEOUT_SECONDS,
+                "--debug",
```

Comment:
How much log this produces and do we need it every time?

Reply:
I think we need it :) (in case something crashes we can investigate in logs and find issues quickly ).  It produces 20 LOC for each installation so it's not big deal.

For clarity here is example:
```java
install.go:178: [debug] Original chart version: ""
install.go:199: [debug] CHART PATH: /home/vsts/work/1/s/packaging/helm-charts/helm3/strimzi-kafka-operator
client.go:128: [debug] creating 1 resource(s)
client.go:128: [debug] creating 1 resource(s)
client.go:128: [debug] creating 1 resource(s)
client.go:128: [debug] creating 1 resource(s)
client.go:128: [debug] creating 1 resource(s)
client.go:128: [debug] creating 1 resource(s)
client.go:128: [debug] creating 1 resource(s)
client.go:128: [debug] creating 1 resource(s)
client.go:128: [debug] creating 1 resource(s)
client.go:128: [debug] creating 1 resource(s)
install.go:165: [debug] Clearing discovery cache
wait.go:48: [debug] beginning wait for 10 resources with timeout of 1m0s
client.go:128: [debug] creating 13 resource(s)
wait.go:48: [debug] beginning wait for 13 resources with timeout of 2m0s
ready.go:277: [debug] Deployment is not ready: infra-namespace/strimzi-cluster-operator. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: infra-namespace/strimzi-cluster-operator. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: infra-namespace/strimzi-cluster-operator. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: infra-namespace/strimzi-cluster-operator. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: infra-namespace/strimzi-cluster-operator. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: infra-namespace/strimzi-cluster-operator. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: infra-namespace/strimzi-cluster-operator. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: infra-namespace/strimzi-cluster-operator. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: infra-namespace/strimzi-cluster-operator. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: infra-namespace/strimzi-cluster-operator. 0 out of 1 expected pods are ready
```

Reply:
Okay, it's not the big deal, I just wanted to know if it produces f.e. 200 extra lines per installation. Thanks :)